src/model/constants/index.ts

```diff
@@ -71,7 +71,15 @@ export const Constants = {
   FLATCAR_CONTAINERLINUX_CGROUP_V2_VERSION: '3033.2.0',
 
   // Used to filter cluster labels in api responses
-  RESTRICTED_CLUSTER_LABEL_KEY_SUBSTRING: 'giantswarm.io',
+  RESTRICTED_CLUSTER_LABEL_KEY_SUBSTRINGS: [
+    'giantswarm.io',
+    'kustomize.toolkit.fluxcd.io',
+  ],
```

Comment:
Could we mark these array constants as `readonly string[]`?

Reply:
Fixed. I marked the whole `Constants` object as readonly. It required some type adjustments in a few places. Here is [the commit](https://github.com/giantswarm/happa/pull/3473/commits/61f1e339df8408d8470f8e980c27d0a510a0eec8).